scripts/install

```diff
@@ -75,7 +79,6 @@ pushd $install_dir
 	php$phpversion artisan passport:install -n
 	php$phpversion artisan storage:link -n
 	php$phpversion artisan config:cache -n
-	php$phpversion artisan key:generate -n
```

Comment:
Was this changed wanted ? Just want to be sure :)

Reply:
Yes, now the key is here https://github.com/Thovi98/2FAuth_ynh/blob/77ebeb85ba6c9c7a89a316bc0d65955218a2ba1b/conf/.env.example#L29 and I set the variable at install :)